test_renovate_helper.py

```
@@ -368,5 +370,45 @@ def boom():
+    '''
+    Every other test in here swaps in a FakeApi, so the suite stayed green
+    while ghapi 2.0 turned all these calls into coroutines and the released
+    image died on startup. These poke the real client instead. Nothing here
+    touches the network -- ghapi builds its endpoint groups from bundled
+    metadata, so constructing a client and reading it back is all local.
+    '''
+
+    OPS = [("issues", "list_comments"), ("issues", "create_comment"),
+           ("issues", "update_comment"), ("issues", "delete_comment"),
+           ("repos", "create_commit_status"), ("pulls", "get"),
+           ("users", "get_authenticated")]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)